app/src/main/java/com/malik/covid/network/ApiInterface.kt

```diff
@@ -1,6 +1,13 @@
 package com.malik.covid.network
 
+import com.malik.covid.models.dao.UserDao
+import retrofit2.http.Body
+import retrofit2.http.POST
+
 
 interface ApiInterface {
 
+    @POST("")
+    suspend fun userProfile(@Body userId: String?): UserDao?
```

Comment:
I assume, this will go away again ;)

Reply:
This Pull request is the basic idea to call the API using MVVM architecture, the endpoint will be updated.